docs_user/modules/con_ceph-daemon-cardinality.adoc

```diff
@@ -24,7 +24,8 @@ As a general rule, the number of services that can be migrated depends on the nu
 | osd | mon/mgr/crash      | rgw/ingress |
 ----
 
-With the {dashboard}, and without {rhos_component_storage_file_first_ref} at least four nodes are required. The {dashboard} has no failover:
+With the {dashboard}, and without {rhos_component_storage_file_first_ref} at
+least four nodes are required. The {Ceph} dashboard has no failover:
```

Comment:
```suggestion
least 4 nodes are required. The {Ceph} dashboard has no failover:
```
This is a style nit. You can use the word or numeral below 10, but we should be consistent. The next sentence uses "5."